gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -173,6 +175,53 @@ def _assert_variants_equal(self, actual, expected):
         sorted(expected),
         sorted(actual))
 
+  def _get_invalid_file_contents(self):
+    """Gets sample invalid files contents.
+
+    Returns:
+       A `tuple` where the first element is contents that are invalid because
+       of record errors and the second element is contents that are invalid
+       because of header errors.
+    """
+    return ([
```

Comment:
nit: I think it's more clear if you make two lists and name them (e.g. malformed_vcf_records, malformed_header_lines) and return (malformed_vcf_records, malformed_header_lines). I think the code should be self-explanatory as much as possible without having to read the comments :)

Reply:
Done.